src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+`--contrast-100` > `--contrast-200` > ··· > `--contrast-1600` > `--contrast-1700`
+
+**Note** that you should never use these swatches to color something, because these swatches will dramatically change their colors and get inverted in dark and light modes.
+
+**Note** that `--color-white` and `--color-black`, which are displayed beside the contrast swatches do not change in dark and light modes. These variables are perfect when used for example in shadows (which remain black even in dark mode) or text colors on colored buttons.
+
+### Dark mode, light mode, usability
+**Note:** don't confuse titles like "dark mode" and "light mode" or "dark/light themes" with these swatches' naming convention. As mentioned above, each swatch -despite its name- already comes with two tones inbuilt into the color system. For example the `--color-red-dark` variable will automatically be displayed as `#d32f2f;` in light mode, and as `#c62828` in dark mode.
+
+<limel-example-dark-light-mode />
+
+The reduced brightness in the dark mode is set with usability in mind, and will help reduce users' eye strain in low-light environments. In a proper design for dark mode, it is not enough to invert the background and text colors. Colors which are bright in light mode, will appear even brighter in dark mode and therefore pop out more from the UI, which can create visual nuisance.
+
+
+## Usage
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus to get a solid blue color, you would write this: `color: rgb(var(--color-blue-default));`,
+or like this to get a blue color with 10% opacity: `color: rgba(var(--color-blue-default), 0.1);`
+
+### Available palettes
+In your web-components, you only need to import one of the palettes. Importing a palette makes all included CSS variables available in that web-component. It all depends on how many colors you need for that specific component. If you need more colors than what is offered in the UI palette, then simply remove that and import one of the other palettes that suits.
+
+Each palette also includes Lime Technologies' brand colors. So no matter which one you import, you will always have access to our brand colors and can use them by their specific CSS variable names.
+
+#### **UI palette** (default)
+Includes 10 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
```

Comment:
```suggestion
Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies brand colors.
```

The reason the apostrophe should be removed here, but not on line 52 for example, is some grammatical thing I don't have words for, just a feeling 😄

Extensive explanation below, skip if you already know or don't care! (I only want to lecture willing subjects 😆)

Compare if we substitute "Volvo" for "Lime Technologies":

> Each palette also includes Lime Technologies' brand colors.
> Each palette also includes Volvo's brand colors.

(implies *all* brand colours are included, so, correct in this case, keep the apostrophe!)

> Each palette also includes Lime Technologies brand colors.
> Each palette also includes Volvo brand colors.  

(I'd say this isn't grammatically incorrect, but it implies only *some* brand colours are includes)

> Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
> Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Volvo's brand colors.

(this clearly doesn't sound right, unless we add an "of": *"+ 11 of Volvo's brand colors"*, but that's just unnecessary and clunky)

> Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies brand colors.
> Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Volvo brand colors.

🎶 😊

Reply:
Hmm.... isn't it like you can have apostrophe for persons, but not objects?

Like:
Kia's brand colors. vs. Volvo brand colors. (Kia stands for Kiarokh not Kia Cars 😜 )

I trust you are right. ;)